avocado/utils/cpu.py

```diff
@@ -550,6 +547,24 @@ def wrap(*args, **kwargs):
     return wrap
 
 
+def lscpu():
+    # Getting the values from lscpu as a dict
```

Comment:
Can you please add python docstring based on [PEP 257](https://peps.python.org/pep-0257) instead of this one line comment? You can inspire by other [methods](https://github.com/avocado-framework/avocado/pull/5772/commits/974d6ebdf9a667b6dd96b45a12c391c53949a0c2#diff-0bf873bc970ebfbd2ce0d991029d110b1b547b14199590a7563d3711b22e9918R534) in this module. The well formatted docstring will be automatically used in documentation.